pygluu/kubernetes/gui.py

```diff
@@ -0,0 +1,934 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+"""
+ License terms and conditions for Gluu Cloud Native Edition:
+ https://www.apache.org/licenses/LICENSE-2.0
+A GUI for installing Gluu Cloud Native Edition.
+"""
+import ipaddress
+import shutil
+import json
+import os
+import base64
+
+from pathlib import Path
+from flask import Flask, jsonify, make_response, render_template, \
+    request, redirect, url_for, send_from_directory
+
+from flask_wtf.csrf import CSRFProtect
+from wtforms.validators import InputRequired, Optional, DataRequired
+from werkzeug.utils import secure_filename
+
+from .common import get_supported_versions, exec_cmd
+from .kubeapi import Kubernetes
+from .forms import LicenseForm, GluuVersionForm, DeploymentArchForm, \
+    GluuNamespaceForm, OptionalServiceForm, GluuGatewayForm, JackrabbitForm, \
+    SettingForm, VolumeTypeForm, CacheTypeForm, CouchbaseMultiClusterForm, \
+    CouchbaseForm, CouchbaseBackupForm, CouchbaseCalculatorForm, \
+    LdapBackupForm, ConfigForm, ImageNameTagForm, ReplicasForm, StorageForm
+
+from .settingdb import SettingDB
+
+app = Flask(__name__, template_folder="templates/gui-install")
+
+cfg = "pygluu.kubernetes.gui_config.DevelopmentConfig"
+app_mode = os.environ.get("FLASK_ENV")
```

Comment:
@rizarsyi `FLASK_ENV` is special env var loaded by Flask app. There are 2 modes: `production` and `development`. If `FLASK_ENV` is not exists, the default is `production`.

The following examples produce same environment:

```
# env is omitted
python app.py
# set to production explicitly
FLASK_ENV=production python app.py
```

Reply:
By default we should load production config (no debug mode, no reloader) when running `pygluu-kubernetes gui-install` or `FLASK_ENV=production pygluu-kubernetes gui-install` command is executed.